src/components/principal/unit/Unit.vue

```diff
@@ -1,17 +1,21 @@
 <template>
-  <div>
+  <div class="unit-locked">
```

Comment:
shoudln't this class be optional?

Reply:
yes it should be optional .. but since integration didn't work I couldn't continued my work(and lock only the required unit).